tests/test_utils.py

```diff
@@ -54,6 +56,13 @@ def test_write_labels(tmpdir):
     assert(os.path.exists(file_path))
 
 
+def test_get_ImageDataGenerator(tmpdir):
+    all_images = utils.get_all_images()
+    image = random.sample(all_images, 1)
+    data_gen = utils.get_ImageDataGenerator(image, (32, 24, 3))
+    assert(isinstance(data_gen, ImageDataGenerator))
+
```

Comment:
You should test that you can get an element from the iterator (and only one) and that the element you get is an image of the shape you selected.